applications/Unity.GrantManager/src/Unity.GrantManager.Application/Intakes/BackgroundWorkers/IntakeSyncWorker.cs

```diff
@@ -39,12 +39,12 @@ public override async Task Execute(IJobExecutionContext context)
             Logger.LogInformation("Executing IntakeSyncWorker...");
 
             var tenants = await _tenantRepository.GetListAsync();
-
+            var numberDaysBack = -2;
```

Comment:
should we make this part of the BackgroundJobsOptions config?

Reply:
Yeah probably though it seems to be a bit of a pain getting config options in

Reply:
Going to setup a config param